### ISSUE TITLE
Error in training the optimal model

### ISSUE THREAD

**Issue by shutiao888** [Adamouization/Breast-Cancer-Detection-Mammogram-Deep-Learning-Publication#2]
I want to know why, when training the optimal model, the loss decreases very slowly on the training log, the accuracy remains unchanged, and ultimately the malignant breast cancer is not recognized at all.Please help me clarify this.The following are some training logs.917/917 [==============================] - 1881s 2s/step - loss: 0.6928 - binary_accuracy: 0.5510 - val_loss: 0.6925 - val_binary_accuracy: 0.5507                                                                                              
Epoch 2/100                                                                                                             
917/917 [==============================] - 99s 108ms/step - loss: 0.6922 - binary_accuracy: 0.5510 - val_loss: 0.6920 - val_binary_accuracy: 0.5507                                                                                             
Epoch 3/100                                                                                                             
917/917 [==============================] - 99s 108ms/step - loss: 0.6917 - binary_accuracy: 0.5510 - val_loss: 0.6915 - val_binary_accuracy: 0.5507                                                                                             
Epoch 4/100                                                                                                             
917/917 [==============================] - 101s 110ms/step - loss: 0.6913 - binary_accuracy: 0.5510 - val_loss: 0.6911 - val_binary_accuracy: 0.5507
Train for 917 steps, validate for 306 steps                                                                             
Epoch 1/50                                                                                                              
917/917 [==============================] - 119s 130ms/step - loss: 0.6879 - binary_accuracy: 0.5510 - val_loss: 0.6881 - val_binary_accuracy: 0.5507                                                                                            
Epoch 2/50                                                                                                              
917/917 [==============================] - 107s 117ms/step - loss: 0.6879 - binary_accuracy: 0.5510 - val_loss: 0.6881 - val_binary_accuracy: 0.5507                                                                                            
Epoch 3/50                                                                                                              
917/917 [==============================] - 107s 117ms/step - loss: 0.6879 - binary_accuracy: 0.5510 - val_loss: 0.6881 - val_binary_accuracy: 0.5507                                                                                            
Epoch 4/50                                                                                                              
917/917 [==============================] - 107s 116ms/step - loss: 0.6879 - binary_accuracy: 0.5510 - val_loss: 0.6881 - val_binary_accuracy: 0.5507
precision    recall  f1-score   support                                                                   
                                                                                                                        
      BENIGN       0.55      1.00      0.71       337                                                                   
   MALIGNANT       0.00      0.00      0.00       275                                                                   
                                                                                                                        
    accuracy                           0.55       612                                                                   
   macro avg       0.28      0.50      0.36       612                                                                   
weighted avg       0.30      0.55      0.39       612
